src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -56,7 +56,7 @@ const BaseHTMLEngineProvider = (props) => {
             systemFonts={_.values(fontFamily)}
         >
             <RenderHTMLConfigProvider
-                defaultTextProps={defaultTextProps}
+                defaultTextProps={{...defaultTextProps, allowFontScaling: false}}
```

Comment:
Shouldn't we add this line along the below lines? @marcaaron or @pecanoro 

https://github.com/Expensify/App/blob/a75744684c7f6e970bb4809e6b12b3c3e84b6890/src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js#L47-L48

Reply:
bump

Reply:
@pecanoro Do you mean we should leave it as it is `defaultTextProps={{...defaultTextProps, allowFontScaling: false}}` allowFontScaling is always 'false' and will never change so there is no need to memoize it?

Reply:
>  we should combine them since it's being used only once, so it makes no sense to extend the array here since it will always be the same value.

@osama256  We should combine them